### PR TITLE
Code for User Story and Created Entity Tests

### PR DESCRIPTION
**Pull request by 39kie**

### CODE REVIEW

File: src/main/view/InputRoomsView.java
Comment (nicholaskohh) [inline src/main/view/InputRoomsView.java:1]: not sure if openFreeViewScreen method should locate in inputroomsview

Comment (xubosen) [reply src/main/view/InputRoomsView.java:1]: not sure either. accept first, ask questions later.

Comment (nicholaskohh) [reply src/main/view/InputRoomsView.java:1]: The majority of the code looks right to me, adding just a button and action listener in view other than the openfreeviewscreen method. We'll accept first then refractor if realize any permission/CA issue.

File: src/test/entity_tests/StairsTest.java
Comment (nicholaskohh) [inline src/test/entity_tests/StairsTest.java:1]: Tests seem accurate.

File: src/main/view/BlueprintSelectionView.java
Comment (nicholaskohh) [inline src/main/view/BlueprintSelectionView.java:1]: Seems accurate, using java swing to display the UI and allowing users to select and view floor blueprints.

File: src/main/interface_adapter/BlueprintViewModel.java
Comment (nicholaskohh) [inline src/main/interface_adapter/BlueprintViewModel.java:1]: seems to properly manage the current blueprint state and notifies listeners of changes

**Review by nicholaskohh — COMMENTED** [review]
Everything looks right.

**Review by nicholaskohh — APPROVED** [review]
Everything looks good.

**Pull request merged**